tests/EvmLookupServer/Test_EvmLookupServer.cpp

```diff
@@ -69,10 +69,17 @@ std::unique_ptr<LookupServer> getLookupServer() {
   return lookupServer;
 }
 
-// Convenience fn only used to test Eth TXs
+// Convenience fn only used to test Eth TXs.
```

Comment:
The transaction and its receipt are stored in this test. The TX receipt needs to have the epoch number in it so that when retrieving it later it doesn't just fail.